src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -203,22 +196,22 @@ private void trim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext ctx, I
     LoadingCache<LogAddress, LogData> dataCache;
```

Comment:
Good catch, I believe this is a bug.

The client is sending a TrimRequest, but its being handled by the fillHole handler.
https://github.com/CorfuDB/CorfuDB/blob/599d9b1881acee1ef47eaf58d98db24d7b1b2665/src/main/java/org/corfudb/runtime/clients/LogUnitClient.java#L259

In order to not overload this patch, I will address it in another one.
Here's the issue; https://github.com/CorfuDB/CorfuDB/issues/260